torch/nn/modules/transformer.py

```diff
@@ -151,14 +152,15 @@ class TransformerEncoder(Module):
         >>> src = torch.rand(10, 32, 512)
         >>> out = transformer_encoder(src)
     """
+    __constants__ = ['norm']
 
     def __init__(self, encoder_layer, num_layers, norm=None):
         super(TransformerEncoder, self).__init__()
         self.layers = _get_clones(encoder_layer, num_layers)
-        self.num_layers = num_layers
```

Comment:
nit: this will be a BC breaking. Prefer not to remove it since we won't claim a BC breaking for this module. Will this block a jitable transformer?

Reply:
I re-added it in the latest commit